multi-node/aws/pkg/config/config.go

```diff
@@ -243,6 +242,18 @@ type Config struct {
 	TLSConfig *CompactTLSAssets
 }
 
+//This functionality now requires AWS API access
+//Separated out from cluster.Config() so that we can only compact
+//assets on command=up
```

Comment:
nit: this is information that should probably be in the commit message. This will quickly lose value as a comment (when is "now")